plugin/src/main/java/org/opensearch/ml/model/MLModelGroupManager.java

```diff
@@ -193,7 +193,9 @@ public void validateUniqueModelGroupName(String name, ActionListener<Boolean> li
     }
 
     private void validateSecurityDisabledOrModelAccessControlDisabled(MLRegisterModelGroupInput input) {
-        if (input.getModelAccessMode() != null || input.getIsAddAllBackendRoles() != null || input.getBackendRoles() != null) {
+        if (input.getModelAccessMode() != null
+            || input.getIsAddAllBackendRoles() != null
+            || !CollectionUtils.isEmpty(input.getBackendRoles())) {
```

Comment:
Seems strange, we better not allow `backend_roles` field show up in user's request instead of passing a empty list, can you change to this behavior?

Reply:
Thanks, think the same. I submit a new commit which add unit test for model group and fix some bug

Reply:
I think we can change this back to `input.getBackEndRoles() != null`, right?

Reply:
I think keep as `CollectionUtils.isEmpty` is safer. Do you see any bug if we keep `CollectionUtils.isEmpty`?

Reply:
I mean we better make the field not showing up when access control is not enabled, user can pass the `backend_roles` field with empty list when using `CollectionUtils.isEmpty`, I prefer to make the field not shown.